lib/stat_tracker.rb

```diff
@@ -20,19 +20,76 @@ def self.from_csv(locations)
 #
 # #------------GameStatistics
 #
-#   def highest_total_score
-#     result = games.max_by do |game|
-#       game.away_goals.to_i + game.home_goals.to_i
-#     end
-#     result.away_goals.to_i + result.home_goals.to_i
-#   end
+  def highest_total_score
+    result = games.max_by do |game|
+      game['away_goals'].to_i + game['home_goals'].to_i
+    end
+    result['away_goals'].to_i + result['home_goals'].to_i
+  end
 #
-#   def lowest_total_score
-#     result = games.min_by do |game|
-#       game.away_goals.to_i + game.home_goals.to_i
-#     end
-#     result.away_goals.to_i + result.home_goals.to_i
-#   end
+  def lowest_total_score
```

Comment:
I like all the small methods!  Good job getting this all written out in such short time!

I told Taylor, and I'll also tell you.  I finally got my coach methods working, and they were passing my tests but NOT the spec harness... I would run all these by that just in case!